user_news/forms.py

```diff
@@ -16,7 +15,7 @@ class OutageForm(ModelForm):
     class Meta:
         model = Outage
         exclude = ['reminder_sent']
-	fields = '__all__'
+        # fields = '__all__'
```

Comment:
Why is this one commented out?

Reply:
setting exclude fields automatically adds all fields except the one in the exclude list, i should probably just remove instead of of commenting it out. The best way to set these is probably to explicitly add all the fields we want to include so we don't inadvertently expose fields that shouldn't go to the front end, but that's probably out of scope for this change